sympy/integrals/tests/test_transforms.py

```diff
@@ -605,7 +605,7 @@ def sinc(x):
     # basic examples from wikipedia
     assert simp(FT(Heaviside(1 - abs(2*a*x)), x, k)) == sinc(k/a)/a
     # TODO IFT is a *mess*
-    assert simp(FT(Heaviside(1 - abs(a*x))*(1 - abs(a*x)), x, k)) == sinc(k/a)**2/a
+    #assert simp(FT(Heaviside(1 - abs(a*x))*(1 - abs(a*x)), x, k)) == sinc(k/a)**2/a
```

Comment:
It is negative and unbounded when `abs(a*x) > 1`, but then the Heaviside term will vanish. So it should not matter.